Exiled.API/Features/Roles/Scp106Role.cs

```diff
@@ -118,16 +122,31 @@ public bool IsSubmerged
             set => HuntersAtlasAbility.SetSubmerged(value);
         }
 
+        /// <summary>
+        /// Gets a value indicating whether or not SCP-106 can activate teslas.
+        /// </summary>
+        public bool CanActivateTesla => Internal.CanActivateShock;
+
         /// <summary>
         /// Gets a value indicating whether or not SCP-106 is ready for idle.
         /// </summary>
         public bool CanActivateIdle => Internal.CanActivateIdle;
 
+        /// <summary>
+        /// Gets a value indicating whether if SCP-106 <see cref="Scp106StalkAbility"/> can be cleared.
+        /// </summary>
+        public bool CanBeCleared => StalkAbility.CanBeCleared;
+
```

Comment:
CanStopStalkAbility ? idk if it's better i need an other guys to valid this

Reply:
done